src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: bold;
+  font-style: normal;
+}
+
+.container {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big {
+  height: 66px;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small {
+  height: 38px;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.form-field {
+  font-family: "Avenir-Heavy", sans-serif;
+  color: #3d4e61;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form-field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-field:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.form-field::placeholder {
+  position: absolute;
+  font-family: "Avenir-Book", sans-serif;
+  color: #3d4e61;
+}
+
+.big::placeholder {
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  margin-top: 20px;
+}
+
+.small::placeholder {
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+  margin-top: 12px;
+}
+
+.icon {
+  position: absolute;
+}
+
+.large {
```

Comment:
The same, please move to general class and use only one modifier for small icon

Reply:
done